src/main/java/com/exactpro/th2/common/schema/message/impl/rabbitmq/AbstractRabbitSubscriber.java

```diff
@@ -239,7 +240,7 @@ protected void handle(String consumeTag, Delivery delivery, T value, Confirmatio
         } catch (Exception e) {
             LOGGER.error("Can not parse value from delivery for: {}", consumeTag, e);
             try {
-                confirmation.confirm();
+                confirmation.reject();
             } catch (IOException ex) {
```

Comment:
Maybe remove this try catch as above reject

Reply:
try-catch can't be removed, reject() throws IOException. The reject() above wrapped into another try-catch

Reply:
the `protected void handle(String consumeTag, Delivery delivery, T value, Confirmation confirmation)` method can throw IOException